ui/keys/keys.go

```diff
@@ -158,3 +159,52 @@ var Keys = KeyMap{
 		key.WithHelp("q", "quit"),
 	),
 }
+
+// Rebind will update our saved keybindings from configuration values.
+func Rebind(universal, issueKeys, prKeys []config.Keybinding) error {
```

Comment:
do you mean to handle `issueKeys` in a later PR?

Reply:
It matches the pr keys stuff so I just didn't handle it yet with the questions I had, but it will be part of this PR.